pkgs/top-level/all-packages.nix

```
@@ -27258,7 +27258,11 @@ with pkgs;
 
   slack-term = callPackage ../applications/networking/instant-messengers/slack-term { };
 
-  singularity = callPackage ../applications/virtualization/singularity { };
+  inherit (
+    callPackage ../applications/virtualization/singularity/packages.nix { }
+  ) singularity-legacy apptainer singularity-ce;
```

```suggestion
  callPackages ../applications/virtualization/singularity { }
    singularity-legacy apptainer singularity-ce;
```
I am not to sure about the syntax but we should use callPackages here.